### ISSUE TITLE
Using best checkpoint with nnUNetv2_find_best_configuration?

### ISSUE THREAD

**Issue by agaldran** [MIC-DKFZ/nnUNet#1491]
Hello Fabian,

Thanks for this fantastic tool, I am enjoying v2 very much. By the way, thanks also for the comments in the code, some of them are really funny to read, and it always reminds me of [this tweet](https://twitter.com/AlexisCompBio/status/1623908738716532737). 

Anyway, I know that how to use best instead of final model **with `nnUNetv2_predict`** has been answered many times, but I would be interested in also getting to choose what checkpoint is loaded when running `nnUNetv2_find_best_configuration`, so that I can check xval performance with ealy-stopping too. 

Tl;dr: could `nnUNetv2_find_best_configuration` have a -chk argument, same as `nnUNetv2_predict`? Thanks!

**Comment by FabianIsensee** [comment]
I just pushed this to the master. Take a look
